website/docs/r/is_subnet_routing_table_attachment.html.markdown

```diff
@@ -0,0 +1,97 @@
+---
+
+subcategory: "VPC infrastructure"
+layout: "ibm"
+page_title: "IBM : subnet_routing_table_attachment"
+description: |-
+  Manages IBM subnet routing table attachment.
+---
+
+# ibm_is_subnet_routing_table_attachment
+Create, update, or delete a subnet routing table attachment resource. For more information, about subnet routing table attachment, see [setting up routing tables](https://cloud.ibm.com/docs/vpc?topic=vpc-using-acls).
+
+**Note:** 
+VPC infrastructure services are a regional specific based endpoint, by default targets to `us-south`. Please make sure to target right region in the provider block as shown in the `provider.tf` file, if VPC service is created in region other than `us-south`.
+
+**provider.tf**
+
+```terraform
+provider "ibm" {
+  region = "eu-gb"
+}
+```
+
+## Example usage
+
+```terraform
+  resource "ibm_is_vpc" "example" {
+		name = "example-vpc"
+	}
+	resource "ibm_is_vpc_routing_table" "example" {
+		vpc   = ibm_is_vpc.example.id
+		name  = "example-rt"
+	}
+
+	resource "ibm_is_subnet" "example" {
+		name                        = "example-subnet"
+		vpc                         = ibm_is_vpc.example.id
+		zone                        = "eu-gb-1"
+		total_ipv4_address_count    = 16
+	}
+
+	resource "ibm_is_subnet_routing_table_attachment" "example" {
+		depends_on    = [ibm_is_vpc_routing_table.example, ibm_is_subnet.example]
```

Comment:
can we remove depends on as we are already referring routing table and subnet resources in the required arguments